Mubashir/style.css

```diff
@@ -0,0 +1,62 @@
+*{
+    padding: 0;
+    /* margin: 0; */
+    box-sizing: border-box;
+    font-family: "Bakbak One","Inter";
+}
+
+.moreicon{
+    height: 30px;
+    width: 60px;
+    margin-left: 20px;
+}
+.navbar{
+    display: flex;
+    justify-content:start;
+    align-items: center;
+}
+.content{
+    width: 100%;
+    color: white;
+    margin-top: 15px;
+    padding: 20px,18px,36px,18px;
+    margin-top: 90px;
+
+}
+.randomtext{
+    background-color: #000000;
+    height: 80px;
+    margin-top: 20px;
+    border-radius: 8px;
+    display: flex;
+    align-items: center;
+}
+.messages{
+    background-color: #000000;
+ height: 60px;
+ margin-top: 20px;
+ border-radius: 8px;
+ display: flex;
+    align-items: center;
+
+}
+.projects{
+    background-color: #000000;
+    height: 40px;
+    margin-top: 20px;
+    border-radius: 8px;
+    display: flex;
+    align-items: center;
+}
+.generalmsg{
+    color: black;
+    display: flex;
+    justify-content: center;
+    align-self: center;
+    font-size:large;
+    font-weight: 900;
+}
+.updates h1{
+    font-weight: 900;
+    margin-left: 575px;
+}
```

Comment:
Remove these properties as they are not aligning the text to the center and also its not responsive : try applying the below directly to the "updates class"

.updates{
font-weight: 900;
position: absolute;
left: 50%;
transform: translateX(-50%);
}